### DISCUSSION TITLE
lootables for elytras in itemframes

### DISCUSSION THREAD

**Discussion by Tizooo** [PurpurMC/Purpur#829] [__Dismissed__] [answered]
option to make the lootable option in paper.yml also refill elytras in itemframes for the end because if u have a border in the end elytras are pretty limited

**Comment by BillyGalbreath** [comment, marked as answer]
This isn't really possible since item frames are not a lootable container like chests. I can't just add the lootable key to it without major changes in the code. This is something a plugin would be better and easier at handling.
